docs/source/release_notes/release-notes.rst

```diff
@@ -50,6 +50,14 @@ develop
     *    - Type
          - Change
 
+    *    - |new|
+         - AtlasDB now implements targeted sweep using a sweep queue.
+           As long as the ``enableSweepQueueWrites`` property of the ``targetedSweep`` configuration is set to true, information about each transactional write and delete will be persisted into the sweep queue.
+           If ``targetedSweep`` is enabled in AtlasDB runtime configurations, background threads will read the persisted information from the sweep queue and delete stale data from the kvs.
+           For more details on targeted sweep, please refer to the :ref:`targeted sweep docs<targeted-sweep>`.
```

Comment:
A short note along the lines of `Consult with the AtlasDB team if you wish to use targeted sweep in addition to, or instead of, standard sweep.` could be useful here - wouldn't want people to pick this up just yet, at least not without (1) us being aware of it, and possibly (2) them being fully aware of the consequences especially if they disable standard BG sweep.